docs/03-github/04-builder.mdx

```diff
@@ -505,6 +517,46 @@ steps:
       sshAgent: ${{ env.SSH_AUTH_SOCK }}
 ```
 
+### Multiple private Github repositories
+
+Github deploy SSH keys are unique per repository. If you have multiple private repositories in your
```

Comment:
Suggestion. The keyword here is `GitHub Deploy Keys`, so we shouldn't put ssh in between there. Perhaps we can change it to something like

```md
[GitHub Deploy Keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/managing-deploy-keys#deploy-keys) are ssh keys that can only be set up on repository level.
```

Reply:
💯